tic_tac_toe/game.py

```diff
@@ -63,22 +76,43 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    for row in range(3):
+        if _is_winning_combination(board,((row,col) for col in range(3)),player):
+            return player
+    for col in range(3):
+        if _is_winning_combination(board,((row,col) for row in range(3)),player):
+            return player
+    if _is_winning_combination(board,((diag,diag) for diag in range(3)), player):
+        return player #0,0 1,1 2,2
+    if _is_winning_combination(board,((diag,2-diag) for diag in range(3)), player):
+        return player #0,2 1,1 2,0
+    return None
```

Comment:
You don't need to explicitly `return None` here. If a function doesn't return anything it is the same thing as returning `None`